lib/ui/view/organization/organization_settings/organization_name_url.dart

```diff
@@ -0,0 +1,150 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_hooks/flutter_hooks.dart';
+import 'package:stacked/stacked.dart';
+import 'package:hng/constants/app_strings.dart';
+import 'package:hng/models/organization_model.dart';
+import 'package:hng/ui/shared/colors.dart';
+import 'package:hng/ui/shared/styles.dart';
+import 'package:hng/ui/shared/text_field.dart';
+import 'package:hng/ui/shared/zuri_appbar.dart';
+import 'package:hng/ui/shared/zuri_loader.dart';
+import 'package:hng/ui/view/organization/organization_settings/organization_settings_view_model.dart';
+
+class OrganizationNameUrl extends HookWidget {
```

Comment:
This is meant to extend to HookViewModelWidget not HookWidget

Reply:
HookViewModelWidget<OrganizationSettingsViewModel>

Reply:
Done